moveit_ros/planning_interface/test/python_move_group_ns.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+import unittest
+import numpy as np
+import rospy
+import rostest
+import os
+
+from moveit_ros_planning_interface._moveit_move_group_interface import MoveGroupInterface
+
+
+class PythonMoveGroupNsTest(unittest.TestCase):
+    PLANNING_GROUP = "manipulator"
+    PLANNING_NS = "test_ns/"
+
+    @classmethod
+    def setUpClass(self):
+        self.group = MoveGroupInterface(self.PLANNING_GROUP, "%srobot_description"%self.PLANNING_NS, self.PLANNING_NS)
+
+    @classmethod
+    def tearDown(self):
+        pass
+
+    def check_target_setting(self, expect, *args):
```

Comment:
same as above

Reply:
Also, why the copy-paste? Is there a reason why you test these functions three times?

Reply:
the magic is all in the initialization of the test; was just ensuring "functionality" aka the robot can plan and execute as demonstrated in MoveGroupCommander tests. so I believe something like these tests should be tested against in each case

Reply:
Makes sense